netbox/netbox/settings.py

```diff
@@ -418,6 +418,9 @@ def _setting(name, default=None):
 ]
 
 # Set up authentication backends
+if not REMOTE_AUTH_ENABLED:
+    # ignore setting if REMOTE_AUTH_ENABLED not set
+    REMOTE_AUTH_BACKEND = 'netbox.authentication.RemoteUserBackend'
```

Comment:
Let's clean up this logic a bit to avoid referencing RemoteUserBackend again. We should only be extending `AUTHENTICATION_BACKENDS` if `REMOTE_AUTH_ENABLED` is true.